### PR TITLE
luci-app-firewall: add some german translations

### PR DESCRIPTION
**Pull request by sch-m**
Signed-off-by: Martin Schiller <ms@dev.tdt.de>

### CODE REVIEW

File: applications/luci-app-firewall/po/de/firewall.po [hunk 1]
```diff
@@ -194,7 +194,7 @@ msgstr "Protokollierung innerhalb der Zone aktivieren"
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/zones.lua:43
 msgid "Experimental feature. Not fully compatible with QoS/SQM."
-msgstr ""
+msgstr "Experimentelles Merkmal. Nicht vollständig kompatibel mit QoS/SQM."
```
Comment (jow-) [inline applications/luci-app-firewall/po/de/firewall.po]: "Experimentelle Funktion(alität)" suits better here imho.

File: applications/luci-app-firewall/po/de/firewall.po [hunk 2]
```diff
@@ -296,7 +296,7 @@ msgstr "Allgemein"
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/zones.lua:51
 msgid "Hardware flow offloading"
-msgstr ""
+msgstr "Hardware-Fluss-Auslagerung"
```
Comment (jow-) [inline applications/luci-app-firewall/po/de/firewall.po]: "Hardwarebeschleunigte Flusskontrolle"

File: applications/luci-app-firewall/po/de/firewall.po [hunk 3]
```diff
@@ -582,15 +582,15 @@ msgstr "Schreibe um auf Quell-%s, %s"
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/zones.lua:42
 msgid "Routing/NAT Offloading"
-msgstr ""
+msgstr "Routing/NAT-Auslagerung"
```
Comment (jow-) [inline applications/luci-app-firewall/po/de/firewall.po]: "Routing/NAT-Beschleunigung"

File: applications/luci-app-firewall/po/de/firewall.po [hunk 4]
```diff
@@ -602,15 +602,15 @@ msgstr "SNAT-Port"
...
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/zones.lua:47
 msgid "Software based offloading for routing/NAT"
-msgstr ""
+msgstr "Softwarebasierte Auslagerung für Routing/NAT"
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/zones.lua:46
 msgid "Software flow offloading"
-msgstr ""
+msgstr "Software-Fluss-Auslagerung"
```
Comment (jow-) [inline applications/luci-app-firewall/po/de/firewall.po]: "(Software)beschleunigte Flusskontrolle"

File: applications/luci-app-firewall/po/de/firewall.po [hunk 5]
```diff
@@ -735,11 +735,11 @@ msgstr ""
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/rule-details.lua:342
 msgid "Thursday"
-msgstr ""
+msgstr "Donnerstag"
 
 #: applications/luci-app-firewall/luasrc/model/cbi/firewall/rule-details.lua:362
 msgid "Time in UTC"
-msgstr ""
+msgstr "Zeit in UTC"
```
Comment (jow-) [inline applications/luci-app-firewall/po/de/firewall.po]: "UTC-Zeit" oder "Zeit ist UTC"

**Comment by jow-** [comment]
Added some suggestions which fit better imho.

**Comment by sch-m** [comment]
Thanks, will update the pr.

**Comment by sch-m** [comment]
@jow- Is the patch ok now?
Or did you mean I should integrate your suggestions with the parentheses?

**Comment by jow-** [comment]
Yep, looks fine to me now. Merged, thanks.

**Pull request merged**